packages/eslint-plugin-jest/src/rules/no-disabled-tests.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+import type {EslintContext, CallExpression} from './types';
+
+/* $FlowFixMe */
+const testFunctions = Object.assign(Object.create(null), {
```

Comment:
If you do `indexOf` on an array, lookups are always linear ( O(n) ) on the array. If you use an object, you can look up items in O(1). This doesn't really matter for small arrays that we are using here, but I use this practice for most things. The reason why I'm using `Object.create(null)` is to create a prototype-less object that doesn't have methods like `toString`. Checking `obj['toString']` on `{}` would return true which we don't want here.